### PR TITLE
Features/webpack

### PR DESCRIPTION
**Pull request by Alzubair98**
all the requirements are done
the to do list is ready for the first launch 👍

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <input class="input-bar" type="text" placeholder = "Add to your list" required/>
+            <i class="fa-solid fa-circle-plus"></i>
+        </div> 
+        
+        <div class="list-elements">
+            <!-- <div class="the-elements">
+                <input type="checkbox">
+                <p>go to gym</p>
+            </div>
+            <hr> -->
```
Comment (AmaduKamara) [inline src/index.html:29]: - [ ] Kindly remove these commented codes to make your codebase clean and lean. It's a good practice to not have commented code in your codebase. Commenting is about writing steps, pseudocode in your code, or small documentation in your code for later reference for you or another developer. Therefore, unused codes commented out is really not a comment but makes your codebase dirty. Please remove it 👍
- [ ] Please design your To-Do app just like the interface given in the project brief. The only thing missing is that you should give a white background color to the app. You can see I drew a red box on the wrapper you should have the white background color. 👍 

image

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+        // the code below must be activate when we have more than one js file
+//   optimization: {
+//     runtimeChunk: 'single',
+//   },
```
Comment (AmaduKamara) [inline webpack.config.js:29]: - [ ] Kindly remove this commented CSS code

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Alzubair98,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ Todo list is structured well as required 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: webpack.config.js (hunk 2 above)
Comment (Alzubair98) [reply webpack.config.js:29]: done

File: src/index.html (hunk 1 above)
Comment (Alzubair98) [reply src/index.html]: done

**Review by gtekle — APPROVED** [review]
Hi @Alzubair98 :wave: , 

# STATUS: APPROVED :tada: :tada: :tada:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved-stamp


## Highlights
:heavy_check_mark: Webpack configured correctly
:heavy_check_mark: Used dynamic array for todo tasks
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Clean and organized code

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**